ds/data/processed/ashe_mean_salary/ashe_mean_salary.nuts2.yaml

```diff
@@ -0,0 +1,39 @@
+api_doc_url:
+  Annual Survey of Hours and Earnings resident data (ASHE): https://www.nomisweb.co.uk/sources/ashe
+  Business Register Employment Survey (BRES): https://www.nomisweb.co.uk/sources/bres
+api_type:
+  Annual Survey of Hours and Earnings resident data (ASHE): POST
+  Business Register Employment Survey (BRES): POST
+data_date: 20201808
+description: This indicator captures the mean salary of full time employed residents in a location
+title: Mean salary
+subtitle: Salary data is available at the Local Authority District levels so we had to convert mean salaries into volumes taking into account the number of people employed (transactions undertaken) in a location and aggregate at the NUTS 2 level through a lookup table.
+endpoint_url:
+  Annual Survey of Hours and Earnings resident data (ASHE): https://www.nomisweb.co.uk/api/v01/dataset/NM_30_1.data.csv
+  Business Register Employment Survey (BRES): https://www.nomisweb.co.uk/api/v01/dataset/NM_189_1.data.csv
+framework_group: public_rnd_capability
+is_experimental: True
+is_public: True
+order: [year, nuts_id, nuts_year_spec, value.id]
+query: See script for geography and variable names
+region:
+  type: NutsRegion
+  level: 2
+schema:
+  nuts_id:
+    type: NutsRegion.nuts_id
+  nuts_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    data_type: float
+    format: .3f
```

Comment:
(1) I’d use `type: GBP`, `data_type: int` and no `format` so that instead of `32801.549` we’d get `32,802` (millipounds seems a bit weird :)